web/files/referencedby.go

```diff
@@ -83,5 +83,5 @@ func RemoveReferencedHandler(c echo.Context) error {
 		return wrapVfsError(err)
 	}
 
-	return c.NoContent(http.StatusNoContent)
+	return c.NoContent(http.StatusOK)
```

Comment:
I was able to reproduce but the culprit code was mine: I call `files.RemoveReferenceHandler` — which calls at the end `c.NoContent` — and then I returned `c.JSON(200, nil)`. Since the first call expects no body and the second provides one I get the error I mentionned. I corrected it.